TestStack.Dossier.Tests/BuilderBuildListTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Shouldly;
+using TestStack.Dossier.DataSources.Generators;
+using TestStack.Dossier.Lists;
+using TestStack.Dossier.Tests.Builders;
+using TestStack.Dossier.Tests.Entities;
+using Xunit;
+
+namespace TestStack.Dossier.Tests
+{
+    public class BuilderBuildListTests
+    {
+        [Fact]
+        public void GivenANormalBuilderInstance_WhenCallingIsListBuilderProxy_ThenReturnFalse()
+        {
+            var builder = Builder<Customer>.CreateNew();
```

Comment:
Can we create a separate DTO class for these tests so the test indicate the usage that we intend for the generic builder (non-domain entities).